src/sentry/relay/config/measurements.py

```diff
@@ -0,0 +1,46 @@
+""" Relay configuration related to transaction measurements. """
+
+
+from typing import Sequence, TypedDict
+
+#: The maximum number of custom measurements to be extracted from transactions.
+CUSTOM_MEASUREMENT_LIMIT = 5
+
+
+#: List used to distinguish between user-defined and well-known measurements.
+#: NOTE: This is redundant with `ALL_MEASUREMENT_METRICS`, which can be removed
+#: once all Relay instances understand the new format.
+BUILTIN_MEASUREMENTS = [
+    "app_start_cold",
+    "app_start_warm",
+    "cls",
+    "fcp",
+    "fid",
+    "fp",
+    "frames_frozen_rate",
+    "frames_frozen",
+    "frames_slow_rate",
+    "frames_slow",
+    "frames_total",
+    "lcp",
+    "stall_count",
+    "stall_longest_time",
+    "stall_percentage",
+    "stall_total_time",
+    "ttfb.requesttime",
+    "ttfb",
+]
+
+MeasurementName = str
```

Comment:
this defines an alias -- perhaps you meant to use `NewType` ?

https://youtu.be/9rRYeunzX8c

Reply:
Love the concept, but I _think_ what I want in this case is a simple type alias, to document that the strings in `knownMeasurements` are measurement names. Let me know if you consider this bad style and would prefer putting this information in a docstring.

Reply:
I would just use `str` -- aliases imo are only useful when you've got a complicated type you want to shorten